Getting-Started.md

```diff
@@ -145,4 +145,56 @@ starting/(re)creating/building/attaching containers for a service.
         nofile:
           soft: 65536
           hard: 65536
-        ```
\ No newline at end of file
+        ```
+     
+   
+
+#### CONNECTION REFUSED
+
+* Indications:  Cannot connect to localhost with ```curl -XGET <elasticsearch-url> -k  ``` 
+
+* Diagnosis: 
+
+    - Check with other version of docker-compose (with/without SecurityGuard), and compose the docker again. (https://github.com/chaoss/grimoirelab-sirmordred#source-code-and-docker)
+
+    - Try changing port number to any number say 9300, and try connecting to localhost. 
+
+* Solution :
+
+    - Check docker logs to see if the error falls under the given categories: 
+    
+	    a. [Low Virtual Memory](https://github.com/ria18405/grimoirelab-sirmordred/blob/master/Getting-Started.md#low-virtual-memory)
+	    
+	    b. [Low File Descriptors](https://github.com/ria18405/grimoirelab-sirmordred/blob/master/Getting-Started.md#low-file-descriptors)
+
+    - In case the error still persists, **Start a fresh environment** 
+
+        - Make sure there is no entry in ```docker ps``` output. To do that, run ```docker-compose down --remove-orphans```
+
+        - Make a fresh installation ```docker system prune -a --volume```, This will clear all containers from the system.
+
+        - Now, execute docker-compose up using the source code (https://github.com/chaoss/grimoirelab-sirmordred#source-code-and-docker 
+
+        - Check connection by connecting to the localhost and later try executing micromondred by setup.cfg and project.json. The data should be visible through the dashboard.
+
+
+#### RATE LIMIT EXHAUSTED
+
+* Indication: While executing perceval, see error message ```RuntimeError: Rate limit exhausted.; 3581.0 seconds to rate reset```
```

Comment:
Maybe it's better to avoid naming Perceval, for instance:
```
* Indication: While executing the collection phase, see error message ```RuntimeError: Rate limit exhausted.; ... seconds to rate reset```

* Solution : Enable the sleep-for-rate parameter in the corresponding section within the setup.cfg
```

I would an example, github or gitlab could work. WDYT?

Reply:
Yeah, right. I used this because I faced an error there. I'll make it generic by changing it. 

Thanks